stable/prometheus/values.yaml

```diff
@@ -1,8 +1,14 @@
+rbac:
+  create: false
+
 alertmanager:
   ## If false, alertmanager will not be installed
   ##
   enabled: true
 
+  # Defines the serviceAccountName to associate with a role when rbac is enabled.
+  serviceAccountName: default
```

Comment:
Pods only run as `.Values.serviceAccountName` if `rbac.create=false`. In that case, none of the RBAC resources are created and it's up to the user to configure permissions externally.